js/objects/tests/test-button-system-commands-with-preload.js

```diff
@@ -29,7 +29,7 @@ describe('Button System Command Tests', () => {
             assert.exists(foundEl);
             assert.exists(foundEl.model);
         });
-        it('Can get card view by its id', () => {
+        it.skip('Can get card view by its id', () => {
```

Comment:
why would this test need to be skipped?

Reply:
you're right it doesn't need to be skipped i just need to change the logic not to depend on the hard-coded 2 value which depended on the old setup. i can fix that up

Reply:
Fixed here: https://github.com/dkrasner/Simpletalk/pull/37/commits/de96a2c0355648cf56d519818d213fc9b174f2bf